tests/transports-integrations/tests/integrations/test_openai.py

```
@@ -0,0 +1,411 @@
+        chunks = []
+        for chunk in stream:
+            if chunk.choices[0].delta.content:
+                chunks.append(chunk.choices[0].delta.content)
+
+        assert len(chunks) > 0, "Should receive streaming chunks"
+        full_content = "".join(chunks)
+        assert len(full_content) > 0, "Streaming content should not be empty"
+
+
+# Additional helper functions specific to OpenAI
+def convert_to_openai_tools(tools: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)